scripts/stringUpdateScript.js

```diff
@@ -0,0 +1,37 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+// This script creates the i18n/sample template used for adding/updating i18n files
+const fse = require("fs-extra");
+const path = require("path");
+
+// Clean out i18n samples directory
+fse.emptyDirSync("./i18n/sample");
```

Comment:
Is it bad to be doing this? Not sure if we expect users might create files in that folder which should be preserved.

Reply:
Developers needs to understand how nls works. If we do not clean the entire dir then it will be hard to catch and remove the files for renamed modules.

Reply:
@tjohnsonBCM  Is there a specific use case you have in mind for when developers would want to manually create files in i18n/sample? In my mind, the contents of i18n/sample should be automatically generated based on wherever the `localize` function is called, and developers should not have to touch anything in that folder. Therefore, it shouldn't be detrimental to clean out i18n/sample before generating an updated set of files for it.

Reply:
No I didn't have a specific use case in mind, was just unsure since the i18n/sample folder wasn't being cleaned in this script previously. Thanks for the explanation, it confirms that emptying the contents of the folder is the desired behavior.